collections/_posts/2023-01-04-Transfer-Learning.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: review
+title: "What is being transferred in transfer learning?"
+tags: pre-training transfer-learning large-scale
+author: "Julia Puig"
+cite:
+    authors: "Behnam Neyshabur, Hanie Sedghi and Chiyuan Zhang"
+    title:   "What is being transferred in transfer learning?"
+    venue:   "NeurIPS 2020"
+pdf: "https://arxiv.org/abs/2008.11687"
```

Comment:
I would link to the published NeurIPS version rather than the arXiv one, since NeurIPS papers are made public :wink: 
```suggestion
pdf: "https://proceedings.neurips.cc/paper/2020/hash/0607f4c705595b911a4f3e7a127b44e0-Abstract.html"
```